packages/desktop/electron/lib/errorPreload.js

```diff
@@ -4,7 +4,7 @@ const { version } = require('../../package.json')
 contextBridge.exposeInMainWorld('error', {
     getData: () =>
         ipcRenderer.invoke('error-data').then((data) => ({
-            iconPath: './assets/logos/firefly_logo.svg',
+            iconPath: './assets/logos/lightmode/logo_iota.svg',
```

Comment:
This logo does not exist. If a new one is going to be added in the future, I would put a `TODO` comment, if not, I would add one that already exists as `logo_iota_full.svg`

Reply:
forgot to push it :pray: